devicon.json

```
@@ -7758,6 +7758,29 @@
         "color": "#ff3e00",
         "aliases": []
     },
+    {
+        "name": "swagger",
+        "altnames": [
+            "Swagger"
+        ],
+        "tags": [
+            "development"
+        ],
```

Some suggestions of tag.
```suggestion
        "tags": [
            "development",
            "software",
            "tool"
        ],
```